google_fastly_waf/variables.tf

```diff
@@ -78,6 +78,16 @@ variable "service_account" {
   default = null
 }
 
+variable "log_sampling_percent" {
+  type    = string
+  default = "10"
```

Comment:
Does this count as a breaking change since it's not `100`? Also semantically it feels like this should be a number but I get that it's being interpolated into a string.